base/show.jl

```diff
@@ -82,17 +82,10 @@ haskey(io::IOContext, key) = haskey(io.dict, key)
 haskey(io::IO, key) = false
 getindex(io::IOContext, key) = getindex(io.dict, key)
 getindex(io::IO, key) = throw(KeyError(key))
-_limit_output = nothing # TODO: delete with with_output_limit deprecation
 function get(io::IOContext, key, default)
```

Comment:
that would make sense now, yeah - wasn't worth going through the ci queue again for though